client/commands.cc

```diff
@@ -402,6 +403,37 @@ command_get_xfiles(DBus::Connection& conn, const string& config_name, unsigned i
 
 
 void
+command_prepare_quota(DBus::Connection& conn, const string& config_name)
+{
+    DBus::MessageMethodCall call(SERVICE, OBJECT, INTERFACE, "PrepareQuota");
+
+    DBus::Hoho hoho(call);
```

Comment:
We just talked about this, but just for the record: "Hoho", "hoho", "Hihi", "hihi" are not good names.
Since this is about DBUS communication, I'd suggest something like "request" and "reply".